runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersListView/UsersListEdit/UsersListEdit.types.ts

```diff
@@ -0,0 +1,25 @@
+import { IUser } from 'runbotics-common';
+
+export interface UsersListEditDialogProps {
+    open: boolean;
+    openDeleteTab: () => void;
+    onClose: () => void;
+    userData: IUser;
+};
+
+export interface IFormValidationState {
```

Comment:
having `I` as an interface prefix is an antipattern
https://youtu.be/qA65QjWCl60?si=oN30dTAPAXAvql-B&t=164